lib/phases/reboot.js

```diff
@@ -63,6 +63,7 @@ Reboot.prototype.reboot = function() {
       return phase._restart();
     })
     .then(function(time) {
+      phase.setup();
```

Comment:
LOL you're right! It's all sync/no-return code. OK, nothing to see here.

(Off-topic: you can still return non-promise-generating code from a `.then` and it will work just fine.)